cli/internal/cmd/configgenerate.go

```diff
@@ -13,14 +13,14 @@ import (
 	"github.com/edgelesssys/constellation/v2/cli/internal/cmd/pathprefix"
 	"github.com/edgelesssys/constellation/v2/internal/attestation/variant"
 	"github.com/edgelesssys/constellation/v2/internal/cloud/cloudprovider"
-	"github.com/edgelesssys/constellation/v2/internal/compatibility"
 	"github.com/edgelesssys/constellation/v2/internal/config"
+	"golang.org/x/mod/semver"
+
```

Comment:
Please remove the space so that `"golang.org/x/mod/semver"` is placed at the old location at the bottom of the list by the formatter.